aws-elasticbeanstalk-terraform/input.tf

```diff
@@ -0,0 +1,4 @@
+locals  {
+    OP_LETSENCRYPT_DOMAIN = "1password-scimbridge-test.us-east-1.elasticbeanstalk.com"
```

Comment:
It would be nice to have a few details here (for both variables) - does this have to match anything?